web/docs/connectors/udp.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_custom_props:
+  connector:
+    loader: true
+    saver: true
+---
+
+# udp
+
+Loads bytes from and saves bytes to a UDP socket.
+
+## Synopsis
+
+Loader:
+
+```
+udp [-c|--connect] [-n|--insert-newlines] <endpoint>
+```
+
+Saver:
+
+```
+udp [-n|--insert-newlines] <endpoint>
+```
+
+## Description
+
+The `udp` connector supports UDP sockets. The loader reads blocks of
+bytes from the socket, and the saver writes them to the socket.
+
+The loader defaults to creating a socket in listening mode. Use `--connect` if
+the loader should initiate the connection instead.
+
+When you have a socket in listening mode, use `0.0.0.0` to accept connections on
+all interfaces. The [`nics`](../operators/nics.md) operator lists all all
+available interfaces.
+
+### `<endpoint>`
+
+The address of the remote endpoint to connect to when using `--connect`, and the
+bind address when using `--listen`.
+
+### `-c,--connect` (Loader)
+
+Connect to `<endpoint>` instead of listening at it.
+
+### `-n,--insert-newlines` (Saver, Loader)
+
+Append a newline character (`\n`) at the end of every datagram.
+
+This option comes in handy in combination with line-based parsers downstream,
+such as NDJSON.
+
+## Examples
+
+Import JSON via UDP by listenting on IP address `127.0.0.1` at port `56789`:
+
+```
+from udp://127.0.0.1:56789
+| import
+```
+
+Use a shell to test the UDP loader with netcat:
+
+```bash
+# Shell 1
+tenzir 'from udp://127.0.0.1:56789'
+# Shell 2
+jq -n '{foo: 42}' | nc -u 127.0.0.1 56789
+```
```

Comment:
This looks a lot nicer if you split it into two code blocks that each have a `title="Shell X"` set.

The second example should also use `socat` rather than `netcat`, because netcat is not portable.

Reply:
> This looks a lot nicer if you split it into two code blocks that each have a `title="Shell X"` set.

Will adjust.

> The second example should also use `socat` rather than `netcat`, because netcat is not portable.

There's a bug in `socat` 1.8 that actually prevents this from working on macOS, so I opted for netcat instead.